src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala

```diff
@@ -99,7 +99,8 @@ trait HeadersProcessor extends ToDownloadProcessor with ScorexLogging with Score
         // If we verify transactions, we don't need to send this header to state.
         // If we don't and this is the best header, we should send this header to state to update state root hash
         val toProcess = if (config.verifyTransactions || !(bestHeaderId == h.id)) Seq.empty else Seq(h)
-        ProgressInfo(None, Seq.empty, toProcess, toDownload(h))
+        val toDwnld = toDownload(h)
```

Comment:
Why?

Reply:
@oskin1 just to have explicit parameters naming

Reply:
@kushti, does `toDwnld` really seem more explicit to you than `toDownload(h)`?

Reply:
But it does not provide any new information and make the code more readable.
It might be useful to add some rule about it to our style guide https://github.com/ergoplatform/ergo-scala-style-guide/blob/master/ErgoStyleGuide.md if it makes sense, otherwise, someone will make a reverse change the next time this piece of code will be touched.